tap_harvest/schema.py

```diff
@@ -0,0 +1,56 @@
+import os
+import json
+from singer import metadata
+from tap_harvest.streams import STREAMS
+
+# Reference:
+# https://github.com/singer-io/getting-started/blob/master/docs/DISCOVERY_MODE.md#Metadata
```

Comment:
Reference of what?

Reply:
Removed reference from here as we have added same in function itself.